added information on --cap-lints to deny-warnings antipattern (#58)
rust-unofficial/patterns: anti_patterns/deny-warnings.md

All this conspires to potentially break the build whenever something changes.

Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
longer be used directly. Note that there is a way around this: With the
`--cap-lints=warn` command line argument, all `deny` lint errors are turned into
warnings (there may still be `forbid` lints that will stop compilation, but

> those are a different topic).

Reviewer: Either you should explain the difference between deny and forbid, or not mention forbid at all

Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
longer be used unless the annotation is removed. This is mitigated with
[--cap-lints]. The `--cap-lints=warn` command line argument, turns all `deny`
lint errors into warnings. But be aware that `forbid` lints are stronger than

> `deny` hence the 'forbid' level can not be overridden to be anything lower than

Reviewer 2: ```suggestion
`deny` hence the 'forbid' level cannot be overridden to be anything lower than
```

I am not native English speaker, so I don't know if I am right here.

Reviewer 3: good spot
